success-criteria.md

```diff
@@ -311,10 +286,7 @@ whose operation relies on web advertising:
 The above principles for an interoperable web seek to help advance the mission
 of the W3C as well as further the objectives outlined by the UK CMA "to promote
 competition for the benefit of consumers, both within and outside the UK, to
-make markets work well for consumers, businesses, and the economy." To ensure a
-level playing field, it is particularly important that smaller organizations be
-able to rely on supply chain vendors they require to operate and for the changes
-in web technology to encourage new market entrants.
+make markets work well for consumers, businesses, and the economy." To ensure a level playing field, it is important that smaller organizations be able to rely on supply chain vendors, and to have those vendors operate transparently and with interoperability, in order for those organizations to operate and embrace changes in web technology while encouraging competition within the market.
```

Comment:
This is not the only concern, smaller organizations must also be able to rely on transparency and reliability from these vendors, including that these vendors operate on behalf as publishers another reason why transparency is important. Further, to create a competitive landscape publishers require interoperability as a principle from such vendors, to have standard methods by which to communicate operational preferences. Proposed change to refocus accordingly.